datasets/imagenet_sketch/README.md

```diff
@@ -0,0 +1,1181 @@
+  |920 | traffic light, traffic signal, stoplight|
+  |921 | book jacket, dust cover, dust jacket, dust wrapper|
+  |922 | menu|
+  |923 | plate|
+  |924 | guacamole|
+  |925 | consomme|
+  |926 | hot pot, hotpot|
+  |927 | trifle|
+  |928 | ice cream, icecream|
+  |929 | ice lolly, lolly, lollipop, popsicle|
+  |930 | French loaf|
+  |931 | bagel, beigel|
+  |932 | pretzel|
+  |933 | cheeseburger|
+  |934 | hotdog, hot dog, red hot|
+  |935 | mashed potato|
+  |936 | head cabbage|
+  |937 | broccoli|
+  |938 | cauliflower|
+  |939 | zucchini, courgette|
+  |940 | spaghetti squash|
+  |941 | acorn squash|
+  |942 | butternut squash|
+  |943 | cucumber, cuke|
+  |944 | artichoke, globe artichoke|
+  |945 | bell pepper|
+  |946 | cardoon|
+  |947 | mushroom|
+  |948 | Granny Smith|
+  |949 | strawberry|
+  |950 | orange|
+  |951 | lemon|
+  |952 | fig|
+  |953 | pineapple, ananas|
+  |954 | banana|
+  |955 | jackfruit, jak, jack|
+  |956 | custard apple|
+  |957 | pomegranate|
+  |958 | hay|
+  |959 | carbonara|
+  |960 | chocolate sauce, chocolate syrup|
+  |961 | dough|
+  |962 | meat loaf, meatloaf|
+  |963 | pizza, pizza pie|
+  |964 | potpie|
+  |965 | burrito|
+  |966 | red wine|
+  |967 | espresso|
+  |968 | cup|
+  |969 | eggnog|
+  |970 | alp|
+  |971 | bubble|
+  |972 | cliff, drop, drop-off|
+  |973 | coral reef|
+  |974 | geyser|
+  |975 | lakeside, lakeshore|
+  |976 | promontory, headland, head, foreland|
+  |977 | sandbar, sand bar|
+  |978 | seashore, coast, seacoast, sea-coast|
+  |979 | valley, vale|
+  |980 | volcano|
+  |981 | ballplayer, baseball player|
+  |982 | groom, bridegroom|
+  |983 | scuba diver|
+  |984 | rapeseed|
+  |985 | daisy|
+  |986 | yellow lady's slipper, yellow lady-slipper, Cypripedium calceolus, Cypripedium parviflorum|
+  |987 | corn|
+  |988 | acorn|
+  |989 | hip, rose hip, rosehip|
+  |990 | buckeye, horse chestnut, conker|
+  |991 | coral fungus|
+  |992 | agaric|
+  |993 | gyromitra|
+  |994 | stinkhorn, carrion fungus|
+  |995 | earthstar|
+  |996 | hen-of-the-woods, hen of the woods, Polyporus frondosus, Grifola frondosa|
+  |997 | bolete|
+  |998 | ear, spike, capitulum|
+  |999 | toilet tissue, toilet paper, bathroom tissue|
+</details>
+
+
+### Data Splits
+
+|             |train|
+|-------------|----:|
+|# of examples|50000|
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[More Information Needed]
+
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
```

Comment:
Like my previous comments, feel free to add links to the corresponding sections of [ImageNet-1K](https://huggingface.co/datasets/imagenet-1k).